File: Zig/src/k01/k01.zig
const std = @import("std");

fn question_1() anyerror!void {
    std.log.info("Hello World!　ようこそZig言語の世界へ！", .{});
}

fn question_2() anyerror!void {
    std.log.info("Hello World!", .{});
    std.log.info("ようこそ", .{});
    std.log.info("Zig言語の世界へ！", .{});
}

fn question_3() anyerror!void {
    std.log.info("整数：{}", .{12345});
    std.log.info("実数：{}", .{123.456789});
    std.log.info("文字：{}", .{'A'});
    std.log.info("文字列：{s}", .{"ABCdef"});
}

fn question_4() anyerror!void {
    std.log.info("              ##", .{});
    std.log.info("             #  #", .{});
    std.log.info("             #  #", .{});
    std.log.info("            #    #", .{});
    std.log.info("           #      #", .{});
    std.log.info("         ##        ##", .{});
    std.log.info("       ##            ##", .{});
    std.log.info("    ###                ###", .{});
    std.log.info(" ###       ##    ##       ###", .{});
    std.log.info("##        #  #  #  #        ##", .{});
    std.log.info("##         ##    ##         ##", .{});
    std.log.info(" ##     #            #     ##", .{});
    std.log.info("  ###     ##########     ###", .{});
    std.log.info("     ###              ###", .{});
    std.log.info("        ##############", .{});
}

pub fn k01(num: usize) anyerror!void {
    switch (num) {
        1 => try question_1(),
        2 => try question_2(),
        3 => try question_3(),
        4 => try question_4(),
        else => {},
    }
}

File: PHP/hello_world_again/src/K01.php
<?php

include "IExecutable.php";

class K01 implements IExecutable
{

    public function question_1()
    {
        echo "Hello World!　ようこそPHP言語の世界へ！";
    }

    public function question_2()
    {
        echo "Hello World!\n";
        echo "ようこそ\n";
        echo "PHP言語の世界へ！\n";
    }

    public function question_3()
    {
        echo "整数：" . 12345 . "\n";
        echo "実数：" . 123.456789 . "\n";
        echo "文字：" . 'A' . "\n";
        echo "文字列：" . "ABCdef" . "\n";
    }

    public function question_4()
    {
        echo "              ##\n";
        echo "             #  #\n";
        echo "             #  #\n";
        echo "            #    #\n";
        echo "           #      #\n";
        echo "         ##        ##\n";
        echo "       ##            ##\n";
        echo "    ###                ###\n";
        echo " ###       ##    ##       ###\n";
        echo "##        #  #  #  #        ##\n";
        echo "##         ##    ##         ##\n";
        echo " ##     #            #     ##\n";
        echo "  ###     ##########     ###\n";
        echo "     ###              ###\n";
        echo "        ##############\n";
    }

    public function execute(int $num): void
    {
        switch ($num) {
            case 1:
                $this->question_1();
                break;
            case 2:
                $this->question_2();
                break;
            case 3:
                $this->question_3();
                break;
            case 4:
                $this->question_4();
                break;
        }
    }
}